.github/workflows/release_gem_package.yml

```diff
@@ -0,0 +1,41 @@
+# Reference Workflow: https://github.com/actions/starter-workflows/blob/main/ci/gem-push.yml
+name: Release Gem Package
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'lib/datum/version.rb'
+
+env:
+  BUNDLE_RUBYGEMS__PKG__GITHUB__COM: ${{ secrets.BUNDLE_RUBYGEMS__PKG__GITHUB__COM }}
```

Comment:
**Question**

@kyc-tm, could you try remove this line and release a test version?
```suggestion
```

Feel we may not need this line, due to we don't have any packages from syntax-enigeering github in below file
https://github.com/syntax-engineering/datum/blob/kc_package_release/datum.gemspec#L6-L23

Reply:
Currently https://github.com/syntax-engineering/datum/ is a public repo, and `BUNDLE_RUBYGEMS__PKG__GITHUB__COM` only allow syntax-enigeering github private repos, which I feel we don't need this line for datum now

Reply:
tested out here: https://github.com/syntax-engineering/datum/pkgs/rubygems/datum/101626310

possible that it's leftover from Acuity (since acuity needs the ENV for fabric)

Reply:
Yeah, good point, thank you!